test/e2e/airgap.go

```diff
@@ -0,0 +1,20 @@
+package e2e
```

Comment:
I think we need the e2e build tags here and the linter should stop complaining:

```suggestion
//go:build e2e

package e2e
```